plugin/yaml/plugin.go

```diff
@@ -100,15 +100,15 @@ func NewPluginSpec(pluginPath string) (*PluginSpec, error) {
 		return nil, err
 	}
 	var plugin PluginSpec
-	if err := yaml.UnmarshalStrict(pluginBytes, &plugin); err != nil {
+	if err := yaml.Unmarshal(pluginBytes, &plugin); err != nil { // TODO: check if strict marshal is required
```

Comment:
why was this removed ? @sbchaos

Reply:
New plugin model has few items removed, so made it less strict for migration.